content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_basic.rst

```diff
@@ -0,0 +1,200 @@
+   case, a |PO| is **always** created, even if there is enough stock on-hand.
+
+Process PO
+----------
+
+If a |PO| was created in the previous step, navigate to it by clicking the :guilabel:`Purchase`
+smart button at the top of the |SO|, or by going to :guilabel:`Purchase app --> Orders --> Purchase
+Orders`, and selecting the |PO|. Then, click :guilabel:`Confirm Order` to confirm it, and move on to
+the next step.
+
+If a |PO| was not created in the previous step, do so now by navigating to :menuselection:`Purchase
+app --> Orders --> Purchase Orders`, and clicking :guilabel:`New`.
+
+Begin filling out the |PO| by selecting a subcontractor from the :guilabel:`Vendor` drop-down menu.
+In the :guilabel:`Products` tab, click :guilabel:`Add a product` to create a new product line.
+Select a subcontracted product in the :guilabel:`Product` field, and enter the quantity in the
+:guilabel:`Quantity` field. Finally, click :guilabel:`Confirm Order` to confirm the |PO|.
+
+When a |PO| is confirmed for a product manufactured using basic subcontracting, a receipt or
+dropship order is automatically created, and can be accessed from the corresponding
+:guilabel:`Receipt` or :guilabel:`Dropship` smart button that appears at the top of the |PO|.
+
+.. figure:: subcontracting_basic/subcontractor-po.png
+   :align: center
+   :alt: A PO for a basic subcontracting product, with a Receipt smart button at the top of the page.
+
+   A PO for a basic subcontracting product, with a Receipt smart button at the top of the page.
+
+Process receipt or dropship order
+---------------------------------
+
+Once the subcontractor has finished manufacturing the product, they either ship it to the
+contracting company, or dropship it to the end customer, depending on how the product was
+:ref:`configured <manufacturing/workflows/subcontracting_basic/product-config>`.
+
+Process receipt
+~~~~~~~~~~~~~~~
+
+If the subcontractor ships the finished product to the contracting company, once it has been
+received, navigate to :menuselection:`Purchase app --> Orders --> Purchase Orders`, and select the
+|PO|.
+
+Click the :guilabel:`Receive Products` button at the top of the |PO|, or the :guilabel:`Receipt`
+smart button at the top of the page, to open the receipt. Then, click :guilabel:`Validate` at the
+top of the receipt to enter the product into inventory.
+
+Process dropship order
+~~~~~~~~~~~~~~~~~~~~~~
+
+If the subcontractor dropships the product, once they have sent it, navigate to
+:menuselection:`Purchase app --> Orders --> Purchase Orders`, and select the |PO|.
+
+Select the :guilabel:`Dropship` smart button at the top of the page to open the dropship order, and
+click :guilabel:`Validate` at the top of the order to confirm that the product has been sent to the
+customer.
+
+Process delivery order
+----------------------
+
+If the subcontracting workflow was started by a customer |SO|, and the finished product was **NOT**
```

Comment:
```suggestion
If the subcontracting workflow was started by a customer |SO|, and the finished product was **not**
```
i'm not sure full capitalization is totally necessary here, so I'm suggesting a lower-case interpretation, while still maintaining its bold formatting -- because it is a necessary/vital element to the sentence.